### ISSUE TITLE
:sparkles: Add task endpoint that provides task queue summary. 

### ISSUE THREAD

**Issue by jortel** [konveyor/tackle2-hub#641]
Add task queue report (counts) needed for the _task bell_.  Probably generally useful to determine how _busy_ the task manager is.

**/tasks/report/queue**
report:
- total (int) - _total number of tasks scheduled_
- ready (int) _number of tasks ready to run_
- postponed (int) _number of postponed tasks_
- quotaBlocked (int) _number of tasks with pods blocked by resource quota_
- pending (int) - _number of tasks with pods created awaiting node scheduler_
- running (int) _number of tasks with running pods_

states that define _queued_:
- Ready
- Postponed
- QuotaBlocked
- Pending
- Running

Filter on:
- kind?
